authors/filip-srdoc/15-porezi/hr_HR/final.md

```diff
@@ -0,0 +1,107 @@
+Prema [FATF-u][fatf] (Financial Action Task Force) virtualni novac obilježavaju klasične funkcije novca koje kažu da je novac, pa tako i onaj virtualni, sredstvo razmijene, mjera vrijednosti te služi za pohranu vrijednosti. No, postoji razlika između klasičnog [fiat][fiat] novca i virtualnog novca. Naime, fiat novac je službeno sredstvo plaćanja u nekoj državi kojeg izdaje centralna banka te države za razliku od virtualnog novca koji nigdje nije prihvaćen kao službeno sredstvo plaćanja i koji nastaje rudarenjem o kojem više možete doznati [ovdje][bc].
+
+Virutalni se novac se razlikuje od elektroničkog novca koji predstavlja digitalnu inačicu prave fiat valute, no ta digitalna valuta može predstavljati i virtualnu i fiat valutu. Najpoznatija digitalna valuta na svijetu danas je bitcoin. Budući da smo o tehničkoj strani bitcoina pisali [ovdje][btc], u ovom članku ćemo se posvetiti porezno-pravnom aspektu.
```

Comment:
- Virtualni
- dvaput se
- Konfuzna rečenica. Spominje se virtualni, digitalni, elektronički i fiat u jednom.
- Ako je najpoznatija digitalna valuta bitcoin, zašto je onda gore spomenuto da se virtualne valute rudare? Koja je razlika digitalne i virtualne? Konfuzno.